app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -707,9 +701,7 @@ class HomeFragment : Fragment(), CoroutineScope, AccountObserver {
     private fun animateTopCollection() {
         launch {
             delay(ANIM_SCROLL_DELAY)
-            val tabsSize = requireComponents.core.sessionManager.sessions.filter {
-                (activity as HomeActivity).browsingModeManager.isPrivate == it.private
-            }.size
+            val tabsSize = getListOfTabs().size
```

Comment:
See above re:
```kotlin
val tabSize = sessionManager.sessions.size
```

If you do this trick another time, consider making it into a calculated property of `HomeFragment`.